docs/foundations/icons/code.md

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
how do we plan on navigating to this page? or is developing a pattern for pages nested more than 1 layer deep TBD?